### PR TITLE
Wine_quality dataset trained

### PR DESCRIPTION
**Pull request by NitiKaur**
Objectives:-
1.To train and test classification model using Sklearn and compare them.
2.To preprocess the data, perform EDA and perform Hyperparameter tuning wherever necessary.

My work:-
I have used the 'Wine Quality' dataset for task 1.I have used SVM, KNN and Logistic Regression Algorithm for predicting if a particular wine brand is recommended or not.I achieved the objective of preprocessing the data, performing EDA and Hyperparameter tunung along with computing an evaluation metric and also plotting graphs for visualising the data. .Since the Recommend Feature had textual data, I converted it into Categorical data so as to ease the computability. I got the accuracy in the different models as follows-
SVM                 -  80.1% 
KNN                 -  86%
Logistic Regression -  99.5%
Since I got less accuracy with the KNN algorithm, I performed Hyperparameter-tuning to compute the best p-value and leaf size.
I have also visualised the data using the seaborn library plotting various features and then analysing the data henceforth. Since Quality had a linear relation with the recommend feature I dropped the feature in the linear SVM algorithm.

@dzeber and @mlopatka please review it once.Thank You!

### CODE REVIEW

**Comment by mlopatka** [comment]
@NitiKaur it looks like you accidentally deleted the notebook.
There is no code to review. Perhaps you switched branches by accident or forgot to track the new version of your notebook before pushing new commits.

**Comment by mlopatka** [comment]
More than 5 days inactive; closing out this PR.
If this is still work in progress, feel free to reopen and flag me.
